keyboards/basekeys/slice/rev1_rgb/rev1.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
```

Comment:
this file should match the folder name rev1_rgb.h

Reply:
Complete